### ISSUE TITLE
Is this method also applicable for fer2013 data set? Why did the author not verify it on the most common fer2013 data set

### ISSUE THREAD

**Issue by 1chenchen22** [yaoing/DAN#34]

**Comment by 1chenchen22** [comment]
The obtained accuracy and loss of Epoch 10/10, Loss: 0.6809, Validation Accuracy: 0.5688, when fine-tuning the DAN model on the Fer2013 dataset and loading the weights of rafdb_epoch21_acc0.897_bacc0.8275.pth, are as such. The significant difference in performance between different datasets can be attributed to several factors. It is possible that the data preprocessing and handling might not have been optimized for the Fer2013 dataset. Additionally, the introduction of the ImbalancedDatasetSampler class to address class imbalance might not have yielded a significant improvement in this case.
